content/_data/changelogs/weekly.yml

```diff
@@ -14565,6 +14565,74 @@
   # pull: 6313 (PR title: Clarify that the changelog entry should be in the imperative mood)
   # pull: 6314 (PR title: Bump script-security from 1138.v8e727069a_025 to 1140.vf967fb_efa_55a_)
 
+  - version: '2.338'
+    date: 2022-03-06
+    changes:
+      - type: rfe
+        category: developer
+        pull: 6317
+        authors:
+          - glasswalk3r
+          - 'TODO: alceu.freitas (alceu.freitas@semantix.com.br)'
+        pr_title: Refactor `translation-tool.pl`
+        message: |-
+          Developer: Add a <code>--debug</code> option to the translation tool to help in troubleshooting.
+           Developer: Improve the readability of the translation tool command-line interface (CLI).
+      - type: TODO
+        category: developer
+        pull: 6329
+        authors:
+          - dependabot[bot]
+        pr_title: Bump Remoting from 4.12 to 4.13
+        message: |-
+          Allow agent reconnects on Java 11 and websocket ([JENKINS-66446](https://issues.jenkins.io/browse/JENKINS-66446) - [PR-514](https://github.com/jenkinsci/remoting/pull/514) ). Prevent infinite loop in case of closed SSL connection ([JENKINS-67928](https://issues.jenkins.io/browse/JENKINS-67928) - [PR-513](https://github.com/jenkinsci/remoting/pull/513) ) Upgrade from  Remoting 4.12 to [4.13](https://github.com/jenkinsci/remoting/releases/tag/remoting-4.13).
+      - type: TODO
+        category: developer
+        pull: 6322
+        authors:
+          - dependabot[bot]
+        pr_title: Bump Guava from 31.0.1 to 31.1
+        message: |-
+          Upgrade the [Guava](https://guava.dev/) library from [31.0.1](https://github.com/google/guava/releases/tag/v31.0.1) (released September 27, 2021) to [31.1](https://github.com/google/guava/releases/tag/v31.1) (released February 28, 2022).
```

Comment:
All the related links mentioned in the message need to be placed under the 'references' section, like this:
```
references:
          - pull: 6322
          - url: https://guava.dev/
            title: Guava
          - url: https://github.com/google/guava/releases/tag/v31.0.1
            title: Guava 31.0.1 changelog
          - url: https://github.com/google/guava/releases/tag/v31.1
            title: Guava 31.1 changelog
```
Note: Please add the release dates if you like, I don't know the right place to add it.